config-linux.md

```diff
@@ -107,6 +111,17 @@ Note that the number of mapping entries MAY be limited by the [kernel][user-name
 ]
 ```
 
+## <a name="configLinuxTimeOffset" />Offset for Time Namespace
+
+**`timeOffsets`** (object, OPTIONAL) sets the offset for Time Namespace. For more information
+see the [time_namespaces](time_namespaces.7).
+
+The name of the clock is the entry key.
+Entry values are objects with the following properties:
+
+* **`secs`** *(int64, OPTIONAL)* - is the offset of clock (in seconds) in the container.
+* **`nanosecs`** *(int64, OPTIONAL)* - is the offset of clock (in nanoseconds) in the container.
```

Comment:
I guess this must be smaller than 10^9(?) Do we want to mention the limits and that this is for fratcions of the seconds?

Reply:
It's already explained in the kernel doc: https://github.com/opencontainers/runtime-spec/pull/1151

>   EINVAL An offset-nanosecs value is greater than 999,999,999.